tests/apollo/test_skvbc_dbsnapshot.py

```diff
@@ -192,6 +225,68 @@ async def test_db_checkpoint_cleanup(self, bft_network, tracker):
         assert num_of_db_snapshots == 4
         self.verify_snapshot_is_available(bft_network, 0, old_snapshot_id, isPresent=False)
 
+    @with_trio
+    @with_bft_network(start_replica_cmd_with_operator, selected_configs=lambda n, f, c: n == 7)
+    @verify_linearizability()
+    async def test_create_dbcheckpoint_cmd(self, bft_network, tracker):
+        """
+            sends a createdbCheckpoint command and test for created dbcheckpoints.
+        """
+        bft_network.start_all_replicas()
+        client = bft_network.random_client()
+        skvbc = kvbc.SimpleKVBCProtocol(bft_network, tracker)
+        for i in range(200):
+            await skvbc.send_write_kv_set()
+
+        op = operator.Operator(bft_network.config, client, bft_network.builddir)
+        rep = await op.create_dbcheckpoint_cmd()
+        data = cmf_msgs.ReconfigurationResponse.deserialize(rep)[0]
+        assert(data.success == True)
+
+        """
+        dbcheckpoint will be created on next stable seq num 300.
+        Adding a 10-second delay to ensure that a dummy client request is sent and reaches a stable seq number of 300.
+        """
+        time.sleep(10)
```

Comment:
You can wait for next stable seqNum instead of sleep

Reply:
Fixed